add start9 package

Georges760 opened the pull request:
to be tested

LocoSlug commented:
If appropriate, I would like to suggest adding the config option pplns_ttl_days. Thanks in advance.

Georges760 committed:
Add Start9 (StartOS) package for Hydra-Pool

Bundles hydrapool, Prometheus, and Grafana into a single container
managed by supervisord, packaged as an .s9pk for sideloading onto
StartOS instances with a bitcoind dependency.
